tests/integration/test_simplified_workflows.py

```
@@ -0,0 +1,556 @@
+        dangerous_patterns = [
+            r'\$\{\{.*github\.event\.comment\.body.*\}\}',  # Unquoted user input
+            r'\$\{\{.*github\.event\.issue\.title.*\}\}',  # Unquoted issue title
+            r'eval\s+\$',  # eval with variable
+        ]
+        
+        import re
+        for pattern in dangerous_patterns:
+            matches = re.search(pattern, content)
+            if matches:
+                # Found potentially dangerous pattern - should be quoted or sanitized
+                assert False, f"Potentially unsafe pattern found in {workflow_file}: {matches.group()}"
```

Ruff (0.14.6)

Assertion always fails, replace with `pytest.fail()`

(PT015)